src/xenia/app/profile_dialogs.h

```diff
@@ -54,19 +55,13 @@ class ProfileConfigDialog final : public ui::ImGuiDialog {
  public:
   ProfileConfigDialog(ui::ImGuiDrawer* imgui_drawer,
                       EmulatorWindow* emulator_window)
-      : ui::ImGuiDialog(imgui_drawer), emulator_window_(emulator_window) {
-    memset(gamertag, 0, sizeof(gamertag));
-  }
+      : ui::ImGuiDialog(imgui_drawer), emulator_window_(emulator_window) {}
 
  protected:
   void OnDraw(ImGuiIO& io) override;
 
  private:
-  bool DrawProfileContent(const uint64_t xuid, const uint8_t user_index,
```

Comment:
I figured we'd want "profile badge" thingy to be consistent across different UI areas so I moved this out to xam_ui.cc.